book/website/community-handbook/presenting.md

```diff
@@ -0,0 +1,14 @@
+# Presenting About the Turing Way
+Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 
+
+### Planning a Talk
+Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
```

Comment:
Reminder to fill in the Slack channel name when this has been established

Reply:
@aleesteele did we pick a channel for this, or do we want to create a new one?

Reply:
we could use the community channel as I think that is the one that most talks and opportunities like that are put in now.

Reply:
Sorry I'm late to this! Seems like there are three applicable channels here:
#general : usually used to ask questions that aren't 
#community : where community celebrations, news, and organisational-related
#events : used to promote external events (in the wider open science network, etc.)

Would agree that for now, the community channel makes sense. Am currently reviewing the slack channels, so names may change at some point in the future to account for folk's needs: https://docs.google.com/spreadsheets/d/11MmzROZGpSkzoB01ATOTpklXrIg86eKSm1xUmPZuhlA/edit#gid=547657140

Reply:
Following up on this, I don't think including slack happens naturally within the community. How about using the wording "If you'd like, please post the details in Slack in the #community channel"?